NDTensors/test/ITensors/TestITensorDMRG/dmrg.jl

```diff
@@ -0,0 +1,37 @@
+using Test
+using ITensors
+using NDTensors
+
+function test_dmrg(elt, N::Integer, dev::Function, cut::Float64, no::Float64)
+  # Create N spin-one degrees of freedom
+  sites = siteinds("S=1", N)
```

Comment:
```suggestion
  sites = siteinds("S=1/2", N)
```
Good to test against spin 1/2 just to make the tests a bit faster, since the local dimension will be smaller.